lib/src/api/casting/casting_config.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:bitmovin_player/src/casting_manager.dart';
+import 'package:equatable/equatable.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'casting_config.g.dart';
+
+/// Configures remote playback behavior.
+@JsonSerializable(explicitToJson: true)
+class RemoteControlConfig extends Equatable {
+  const RemoteControlConfig({
+    this.receiverStylesheetUrl,
+    this.customReceiverConfig = const {},
+    this.isCastEnabled = true,
+    this.sendManifestRequestsWithCredentials = false,
+    this.sendSegmentRequestsWithCredentials = false,
+    this.sendDrmLicenseRequestsWithCredentials = false,
+  });
+
+  factory RemoteControlConfig.fromJson(Map<String, dynamic> json) {
+    return _$RemoteControlConfigFromJson(json);
+  }
+
+  Map<String, dynamic> toJson() => _$RemoteControlConfigToJson(this);
+
+  /// A URL to a CSS file the receiver app loads to style the receiver app.
+  /// Default value is `null`, indicating that the default CSS of the receiver
+  /// app will be used.
+  final String? receiverStylesheetUrl;
+
+  /// A map containing custom configuration values that are sent to the remote
+  /// control receiver.
+  /// Default value is an empty map.
+  final Map<String, String> customReceiverConfig;
+
+  /// Whether casting is enabled.
+  /// Default value is `true`.
+  ///
+  /// Has no effect if the `BitmovinCastManager` is not initialized before the
+  /// `Player` is created with this configuration.
```

Comment:
Symbol references should be done using square brackets. Counts for all occurrences in this PR

Reply:
Done